core/src/main/java/com/novoda/noplayer/model/RendererCapability.java

```diff
@@ -0,0 +1,28 @@
+package com.novoda.noplayer.model;
+
+import com.google.android.exoplayer2.RendererCapabilities;
```

Comment:
This can't be exposed here. All exoplayer references need to be moved to the internal package. We shouldn't be exposing this. If we have a `RendererCapability` there should be a class that knows how to transform to it from whatever the underlying player is.